### ISSUE TITLE
Remove link titles for accessibility

### ISSUE THREAD

**Issue by qubist** [Holochain/website#19]
Most of the links on the site have titles like `Link: https://thelink.url/subdomain`. This is really not helpful for accessibility. The title attribute should be removed outright from everything.

Source:

https://developer.paciellogroup.com/blog/2012/01/html5-accessibility-chops-title-attribute-use-and-abuse/

https://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/

**Comment by zippy** [comment]
also updated github links
